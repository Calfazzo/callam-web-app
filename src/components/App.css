.App {
  text-align: center;
  min-width: 300px;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 80px;
}

.App-header {
  background-color: #222;
  /* height: 150px;
  padding: 20px; */
  color: white;
}

.headshot{
  margin: 10px;
}

.App-intro {
  font-size: large;
}

.summary{
  text-align: left;
  padding-top: 10px;
}

@keyframes App-logo-spin {
  from { transform: rotate(0deg); }
  to { transform: rotate(360deg); }
}

.horizontal-line {
  display: block;
  height: 1px;
  border: 0;
  border-top: 1px solid #ccc;
  margin: 1em 0;
  padding: 0;
}

.section-heading {
  border-bottom: 2px #888 solid;
  width: 100%;
  text-align: left;
}

.main-content{
  padding-left: 10px;
  padding-right: 10px;
}

.top-10 {
  margin-top: 10px;
}

li {
  text-align: left;
}