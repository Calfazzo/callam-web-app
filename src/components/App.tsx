import * as React from 'react';
import './App.css';
import { Button, Grid, Row, Col } from 'react-bootstrap';

const logo = require('../static/images/logo.svg');

class App extends React.Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <h1 className="App-title">Welcome to Website</h1>
        </header>
        <Grid>
          <Row>
            <Col xs={12} md={8}>asd</Col>
            <Col xs={6} md={4}>123</Col>
          </Row>
        </Grid>
        <p className="App-intro">
          Awaiting Image is currently under construction.
        </p>
        <Button bsStyle="primary">Primary</Button>
      </div>
    );
  }
}

export default App;
