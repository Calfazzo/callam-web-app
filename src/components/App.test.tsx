/// <reference types="@types/jest" />
import * as React from 'react';
import * as ReactDOM from 'react-dom';
import App from './App';
import * as renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});

it('renders correctly', () => {
  const tree = renderer
    .create(<App />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
