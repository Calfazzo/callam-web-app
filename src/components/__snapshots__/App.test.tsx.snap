// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <h1
      className="App-title"
    >
      Welcome to Website
    </h1>
  </header>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-8 col-xs-12"
      >
        asd
      </div>
      <div
        className="col-md-4 col-xs-6"
      >
        123
      </div>
    </div>
  </div>
  <p
    className="App-intro"
  >
    Awaiting Image is currently under construction.
  </p>
  <button
    className="btn btn-primary"
    disabled={false}
    type="button"
  >
    Primary
  </button>
</div>
`;
